### PR TITLE
Do not check for evmjit, but evmc+cable to see if submodules are checked out)

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ HunterGate(
 project(aleth)
 set(PROJECT_VERSION 1.5.0.dev2)
 
-if (NOT EXISTS ${CMAKE_SOURCE_DIR}/evmjit/.git)
+if (NOT EXISTS ${CMAKE_SOURCE_DIR}/evmc/.git OR NOT EXISTS ${CMAKE_SOURCE_DIR}/cmake/cable/.git)
```
[the remark this answers is not in the document]
Comment (axic) [reply CMakeLists.txt]: In next PRs I am going to remove evmjit and hera.

The tests are only enabled on tests, we could add that.

[the remark this answers is not in the document]
Comment (axic) [reply CMakeLists.txt]: Added tests.

[the remark this answers is not in the document]
Comment (gumb0) [reply CMakeLists.txt]: I'd add jsontests, I think they it's needed to build testeth

File: CMakeLists.txt [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 cmake_minimum_required(VERSION 3.5.1)
 
+if (NOT EXISTS ${CMAKE_SOURCE_DIR}/evmc/.git OR NOT EXISTS ${CMAKE_SOURCE_DIR}/cmake/cable/.git OR (TESTS AND NOT EXISTS ${CMAKE_SOURCE_DIR}/test/jsontests))
```
Comment (axic) [inline CMakeLists.txt]: I think moving this up this early will remove the `TESTS` variable.

I just propose we check here unconditionally for `evmc`, `cmake/cable` and `test/jsontests`.

**Comment by axic** [comment]
@chfast @gumb0 simplified

**Pull request merged**